catalogs/catalog-kafka/src/test/java/com/datastrato/gravitino/catalog/kafka/integration/test/CatalogKafkaIT.java

```diff
@@ -0,0 +1,366 @@
+import com.datastrato.gravitino.integration.test.util.AbstractIT;
+import com.datastrato.gravitino.integration.test.util.GravitinoITUtils;
+import com.datastrato.gravitino.messaging.Topic;
+import com.datastrato.gravitino.messaging.TopicChange;
+import com.datastrato.gravitino.rel.Schema;
+import com.datastrato.gravitino.rel.SchemaChange;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.Config;
+import org.apache.kafka.clients.admin.TopicDescription;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.config.ConfigResource;
+import org.apache.kafka.common.config.TopicConfig;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Tag("gravitino-docker-it")
+public class CatalogKafkaIT extends AbstractIT {
+  private static final Logger LOG = LoggerFactory.getLogger(CatalogKafkaIT.class);
+  private static final ContainerSuite CONTAINER_SUITE = ContainerSuite.getInstance();
+  private static final String METALAKE_NAME =
+      GravitinoITUtils.genRandomName("catalogKafkaIT_metalake");
+  private static final String CATALOG_NAME =
+      GravitinoITUtils.genRandomName("catalogKafkaIT_catalog");
+  private static final String DEFAULT_SCHEMA_NAME = "default";
+  private static final String PROVIDER = "kafka";
+  private static GravitinoMetalake metalake;
+  private static Catalog catalog;
+  private static String kafkaBootstrapServers;
+  private static AdminClient adminClient;
+
+  @BeforeAll
+  public static void startUp() {
+    CONTAINER_SUITE.startKafkaContainer();
+    kafkaBootstrapServers =
+        String.format(
+            "%s:%d",
+            CONTAINER_SUITE.getKafkaContainer().getContainerIpAddress(), DEFAULT_BROKER_PORT);
+    adminClient = AdminClient.create(ImmutableMap.of(BOOTSTRAP_SERVERS, kafkaBootstrapServers));
+
+    createMetalake();
+    createCatalog();
+  }
+
+  @AfterAll
+  public static void shutdown() {
```

Comment:
You don't have the logic to cover the drop catalog and metalake, so you will not meet the issue that Kafka catalog cannot be dropped, you should add this logic.